### PR TITLE
Assurance 1.1.4 release notes

### PR DESCRIPTION
**Pull request by PravinPK**

### CODE REVIEW

File: release-notes/README.md [hunk 1]
```diff
@@ -44,12 +44,20 @@ For more details, see the documentation and release notes in the [aepsdk-react-n
...
 
 ### iOS AEPEdgeConsent 1.0.1
 
 * Updates timestamp in Consent requests to use fractional seconds.
 
+## April 2, 2022
+
+### iOS AEPAssurance 1.1.4
+
+* Fixed a bug that caused Assurance to not connect to a session if your iOS app's info.plist contains property of type date.
```
Comment (calebk1) [inline release-notes/README.md:57]: "... if your iOS app's `info.plist` contains a property of type `date`."

File: release-notes/README.md [hunk 2]
```diff
@@ -44,12 +44,20 @@ For more details, see the documentation and release notes in the [aepsdk-react-n
...
+
+Note: This release pertains to Assurance mobile extension that works with ACPCore
```
Comment (calebk1) [inline release-notes/README.md:59]: I'm not sure what this note means.

**Review by calebk1 — CHANGES_REQUESTED** [review]

**Review by emdobrin — COMMENTED** [review]
@PravinPK can you also include this note to https://aep-sdks.gitbook.io/docs/foundation-extensions/adobe-experience-platform-assurance/adobe-experience-platform-assurance-release-notes ?

File: release-notes/README.md (hunk 2 above)
Comment (PravinPK) [reply release-notes/README.md:59]: @calebk1 
Unlike other iOS extensions where
ACPAnalytics works with ACPCore
and the new swift based 
AEPAnalytics works with AEPCore

Assurance is a little unusual where we have the same name AEPAssurance extension  but

AEPAssurance 1.x.x works with ACPCore
AEPAssuracne 3.x.x works with AEPCore

So this release is an update to AEPAssurance that works with ACPCore

**Review by emdobrin — APPROVED** [review]

**Pull request merged**